docs/reference/content/driver/tutorials/ssl.md

```diff
@@ -201,3 +201,53 @@ ERefGuide.html).
 Some applications may want to force only the TLS 1.2 protocol. To do this, set the `jdk.tls.client.protocols` system property to "TLSv1.2".
 
 Java runtime environments prior to Java 8 started to enable the TLS 1.2 protocol only in later updates, as shown in the previous section. For the driver to force the use of the TLS 1.2 protocol with a Java runtime environment prior to Java 8, ensure that the update has TLS 1.2 enabled.
+
+
+## OCSP
+
+{{% note %}}
+The Java driver cannot enable OCSP by default on a per MongoClient basis.
+{{% /note %}}
+
+### Client-driven OCSP
+
+An application will need to set JVM system and security properties to ensure that client-driven OCSP is enabled:
+
+-  `com.sun.net.ssl.checkRevocation`:
+      When set to `true`, this system property enables revocation checking.
+
+-  `ocsp.enable`:
+      When set to `true`, this security property enables client-driven OCSP.
+      
+To configure an application to use client-driven OCSP, the application must already be set up to connect to a server using TLS. Setting these system properties is required to enable client-driven OCSP.
+
+{{% note %}}
+The support for TLS provided by the JDK utilizes “hard fail” behavior in the case of an unavailable OCSP responder in contrast to the mongo shell and drivers that utilize “soft-fail” behavior.
+{{% /note %}}
+
+### OCSP Stapling
+
+{{% note class="important" %}}
+MongoDB servers running on Red Hat Enterprise Linux 7 do not support OCSP stapling. A Java driver running on Red Hat Enterprise Linux 7 will be able to utilize OCSP stapling.
```

Comment:
@vincentkam is this note required for all drivers?  This seems more like something that should be in the server documentation.

Reply:
This note originally started out as documenting a RHEL7 restriction was clarified to note that it was only a server restriction. I'm fine with removing this note as there's no limitation in the driver itself on RHEL7, but I defer to the Java team as whether this note is worth it: would it be surprising to users to find that stapling works with the driver but not with the server on RHEL7? It's worth noting that Windows and macOS mongods also do not supported stapled OCSP at the moment, so perhaps it's simpler to not document server restrictions?

Reply:
Since there are other environments that do not support stapled OCSP, it is best to leave that list in the server documentation. I have removed the note.